### DISCUSSION TITLE
Frozen videos in iOS app - how to refresh all encoded videos?

### DISCUSSION THREAD

**Discussion by Qhilm** [immich-app/immich#12733] [Q&A]
I have lots of videos that somehow are frozen on the first frame, in the iOS app (I have not tested the android app). Audio is fine, and in the browser on a mac even the video is fine.

I have noticed that if I "refresh encoded videos" on the specific video, it fixes the issue.

Hence my question: how can I run this on all videos?

**Comment by bo0tzz** [comment]
By running "transcode videos" in the jobs panel https://my.immich.app/admin/jobs-status

**Reply by Qhilm** [reply]
But reencoding all videos is massive, it's not what "refresh encoded videos" does, it's instantaneous.

**Reply by bo0tzz** [reply]
It's definitely the same thing. If you're running it for only one asset then it's not too weird for it to be pretty quick.

**Reply by Qhilm** [reply]
ah, I know, I did "refresh encoded videos" on a file which matched the criteria for _not_ being reencoded (resolution was quite low), that's why it was so quick. I originally had settings that caused significantly more videos to be reencoded, compared to my current settings.

I'll reencode all videos and we'll see how long it takes. Thanks.
